Libs/vtkSegmentationCore/vtkSegmentation.cxx

```diff
@@ -47,10 +48,18 @@
 
 // STD includes
 #include <algorithm>
+#include <chrono>
 #include <functional>
 #include <sstream>
 
 const int DEFAULT_LABEL_VALUE = 1;
+const int DEFAULT_SEGMENT_ID_LENGTH = 16;
+
+//----------------------------------------------------------------------------
+// The segment ID randomizer singleton instance.
+// This MUST be default initialized to zero by the compiler and is
+// therefore not initialized here. The classInitialize and classFinalize methods handle this instance.
+static vtkMinimalStandardRandomSequence* vtkSegmentationRandomSequence;
```

Comment:
`vtkSegmentationRandomSequence` name can be mistaken for a class name. It would be better to change the name that reflects that it is a global variable.

Reply:
Fixed by updating to use the macros.